test/testdata.js

```diff
@@ -15,7 +15,15 @@ var TestData = {
         async.series([
             function(cb) {
                 mongoose.model('CraReporters').create(
-                    {'activity_year': '2013', 'respondent_id': '0000000001', 'agency_code': '1', 'respondent_name': 'foo' },
+                    {'activity_year': '2013', 'respondent_id': '0000000324', 'agency_code': '1', 'respondent_name': 'foo' },
```

Comment:
This change in test data is causing test failures.  Always make sure to run the tests before you push changes.